src/shared/services/commandInterpreterHelper.js

```diff
@@ -490,13 +490,19 @@ const availableCommands = [
       }
 
       const url = action.cmd.substr(cmdchar.length + 'play '.length)
+      const anchor = document.createElement('a')
+      anchor.href = url
+      const filenameExtension = anchor.pathname.includes('.')
+        ? anchor.pathname.split('.').pop()
+        : 'html'
```

Comment:
This feel like a hack. Can we do something for formal, like using https://developer.mozilla.org/en-US/docs/Web/API/URL or something?

Reply:
Should I create the URL object with `new URL(url)` instead of `document.createElement('a')`? I agree that it is easier to read, I only used the createElement function because it works in IE.

Reply:
I see. 

The `URL` should be polyfilled so it works in IE as well since we declare that we need support for it in https://github.com/neo4j/neo4j-browser/blob/master/babel.config.js#L19

Good call though, let's change and try it out in IE before next release.